### ISSUE TITLE
Crash when pressing '+' button

### ISSUE THREAD

**Issue by neekless** [nextcloud/android#7878]
### Steps to reproduce
1. Press floating '+' button on bottom right

### Expected behaviour
- crash

### Actual behaviour
- not crash

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
************ CAUSE OF ERROR ************

java.lang.IllegalStateException: The specified child already has a parent. You must call removeView() on the child's parent first.
	at android.view.ViewGroup.addViewInner(ViewGroup.java:5235)
	at android.view.ViewGroup.addView(ViewGroup.java:5064)
	at android.view.ViewGroup.addView(ViewGroup.java:5004)
	at android.view.ViewGroup.addView(ViewGroup.java:4976)
	at com.owncloud.android.ui.fragment.OCFileListBottomSheetDialog.onCreate(OCFileListBottomSheetDialog.java:131)
	at android.app.Dialog.dispatchOnCreate(Dialog.java:419)
	at android.app.Dialog.show(Dialog.java:313)
	at com.owncloud.android.ui.fragment.OCFileListFragment.lambda$registerFabListener$2$OCFileListFragment(OCFileListFragment.java:442)
	at com.owncloud.android.ui.fragment.-$$Lambda$OCFileListFragment$aYkoWPoxzAOahKL5AVjT06NhcnM.onClick(Unknown Source:4)
	at android.view.View.performClick(View.java:7448)
	at android.view.View.performClickInternal(View.java:7425)
	at android.view.View.access$3600(View.java:810)
	at android.view.View$PerformClick.run(View.java:28305)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7660)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140251
Build flavor: generic

************ DEVICE INFORMATION ************
Brand: google
Device: taimen
Model: Pixel 2 XL
Id: RQ1A.210105.003
Product: taimen

************ FIRMWARE ************
SDK: 30
Release: 11
Incremental: eng.dragon.20210118.192928

```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by neekless** [comment]
Nvm saw it's alr been fixed

**Comment by AndyScherzinger** [comment]
And released, just update to 3.14.3 😊
